src/lib/DatePickerInput.tsx

```diff
@@ -170,16 +171,16 @@ const DatePickerInput = <T extends FieldValues>(props: DatePickerInputProps<T>)
               field.onBlur();
             }}
             onChange={(date) => {
-              setSelectedDate(date);
+              setSelectedDate(date ?? undefined);
 
-              const convertedDate = getConvertedDate(date);
+              const convertedDate = getConvertedDate(date ?? undefined);
 
               if (props.onChange) props.onChange(convertedDate);
 
               field.onChange(convertedDate);
             }}
             onClickOutside={(e) => {
-              if (document.getElementById(formGroupId.current)?.contains(e.target as HTMLElement) && !disabled && !formDisabled) {
+              if (document.querySelector(`#${formGroupId.current}`)?.contains(e.target as HTMLElement) && !disabled && !formDisabled) {
```

Comment:
`unicorn/prefer-query-selector` 

I don't understand their reasoning but we also won't have many instances of this. But IMO can be turned off.

Reply:
me neither, i actually wasn't even aware of this selector :D i can also deactivate it? as its only an astethic reason (to match more the jq selector)